src/components/ImageViewer/ImageViewer.tsx

```diff
@@ -33,7 +34,11 @@ const ImageViewer: React.FC<IIIFExternalWebResource> = ({ service }) => {
    * Loads tileSource of current canvas from IIIF image service
    */
   useEffect(() => {
-    if (imageService) openSeadragonInstance?.open(imageService.id);
+    if (imageService) {
+      getInfoResponse(imageService.id).then((tileSource) =>
+        openSeadragonInstance?.open(tileSource),
```

Comment:
Will `tileSource` always have a value here?  What would happen if the shape of `tileSource` is not what we expect, or `undefined` or does OpenSeadragon handle that for us?